front/lib/api/assistant/pubsub.ts

```diff
@@ -45,11 +42,13 @@ export async function postUserMessageWithPubSub(
     content,
     mentions,
     context,
+    jitActions,
   }: {
     conversation: ConversationType;
     content: string;
     mentions: MentionType[];
     context: UserMessageContext;
+    jitActions?: UnsavedAgentActionConfigurationType[];
```

Comment:
Are you sure you need to add a new args ?
I don't have a complete grasp of your PR but I remember when I looked for JIT datasources that we already have AgentConfiguration & actions that are not saved at all (our default agents such as @Dust etc...).

Reply:
💯 if we can avoid adding a new parameter  - but i don't see how, we need to be able to pass it somehow ?

Reply:
I think it would be more efficient to look into it live together. Tomorrow ?